README.md

```diff
@@ -35,7 +44,9 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+// Your answer..
+var andy = new Instructor ({name: Andy,
+description: "Resin Laying Deer Figurine, Gold"})
```

Comment:
`new` does not save the instructor in the database. You need to make sure to also call `.save`